server/app/auth/CiviFormProfileAdapterHelper.java

```diff
@@ -0,0 +1,66 @@
+package auth;
+
+import java.util.Optional;
+import java.util.function.BiFunction;
+import javax.inject.Provider;
+import models.Account;
+import models.Applicant;
+import org.pac4j.core.profile.UserProfile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import repository.UserRepository;
+
+public class CiviFormProfileAdapterHelper {
```

Comment:
suggestion (blocking): please add documentation for this new class

Reply:
Got it, `CiviFormProfileAdapterHelper` is pretty generic. I'd prefer to keep it scoped to just what it's implementing in `threeWayMerge`. Maybe `CiviFormProfileMerger`?

Reply:
Sure sounds good.